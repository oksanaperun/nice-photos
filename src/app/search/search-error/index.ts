export * from './search-error.component';
