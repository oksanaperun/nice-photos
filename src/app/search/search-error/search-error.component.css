h3 {
  font-family: 'Nunito', serif;
  text-align: center;
  color: #f44336;
}
