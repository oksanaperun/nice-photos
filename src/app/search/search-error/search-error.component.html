<h3 i18n>Sorry, something went wrong...</h3>
