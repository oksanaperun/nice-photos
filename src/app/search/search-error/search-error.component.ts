import { Component, ChangeDetectionStrategy } from '@angular/core';

@Component({
  selector: 'app-search-error',
  templateUrl: './search-error.component.html',
  styleUrls: ['./search-error.component.css'],
  changeDetection: ChangeDetectionStrategy.OnPush,
})

export class SearchErrorComponent {}
