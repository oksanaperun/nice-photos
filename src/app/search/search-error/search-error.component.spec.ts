import { TestBed, ComponentFixture } from '@angular/core/testing';
import { SearchErrorComponent } from './search-error.component';

describe('SearchErrorComponent', () => {
  let component: SearchErrorComponent;
  let fixture: ComponentFixture<SearchErrorComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [SearchErrorComponent],
    }).compileComponents();

    fixture = TestBed.createComponent(SearchErrorComponent);
    component = fixture.componentInstance;
  });

  it('should render message', () => {
    expect(fixture).toMatchSnapshot();
  });
});
