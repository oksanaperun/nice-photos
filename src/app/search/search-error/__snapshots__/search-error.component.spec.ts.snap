// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchErrorComponent should render message 1`] = `
<app-search-error>
  <h3>
    Sorry, something went wrong...
  </h3>
  
</app-search-error>
`;
